src/ui/MissingTerm/MissingTerms.ts

```diff
@@ -189,6 +189,16 @@ export class MissingTerms extends Component {
     return foundStar || foundQuestionMark;
   }
 
+  private constainsApostrophe(term: string): boolean {
+    if (term.length > 1) {
+      return false;
+    }
+    const query = this.queryStateModel.get('q');
+    const regxApostrophe = RegExp(`'${term}|${term}'`);
```

Comment:
What you need to know is that from the API call, we don't receive term with `'` in them. Words with `'` are treated as two different term 
For example,
We could have the term `efile's` in a query. if the term is not present, we could receive only `efile`, only `s` or both at the same time  

Right now, I only check if the missing term **of length 1** has an `'` before or after to hide it. I do this because I think we want to include the term `efile` but including only `s` doesn't make much sense.

As for the whitelist, It wouldn't solve the problem, we need to support a lot of languages and I don't think blocking `'` is an option. It uses a lot in French and English so removing it would be bad.

Reply:
Ok, what about the regex itself `RegExp(`'${term}|${term}'`);`, could you give me an example of how it works?

Reply:
if my term is `s` and my query is `efile's` the regex will try to find if `'s` or `s'` is present the query.
Since it is present, I will not display this term as a missing term.